src/EditorFeatures/CSharpTest/SimplifyTypeNames/SimplifyTypeNamesTests.cs

```diff
@@ -4177,10 +4234,10 @@ private async Task TestWithPredefinedTypeOptionsAsync(string code, string expect
             SingleOption(CodeStyleOptions.PreferIntrinsicPredefinedTypeKeywordInMemberAccess, true, NotificationOption.Error),
             SingleOption(CodeStyleOptions.PreferIntrinsicPredefinedTypeKeywordInDeclaration, this.offWithSilent, GetLanguage()));
 
-        private IDictionary<OptionKey, object> PreferImplicitTypeWithSilent => OptionsSet(
-            SingleOption(CSharpCodeStyleOptions.VarElsewhere, onWithSilent),
-            SingleOption(CSharpCodeStyleOptions.VarWhenTypeIsApparent, onWithSilent),
-            SingleOption(CSharpCodeStyleOptions.VarForBuiltInTypes, onWithSilent));
+        private IDictionary<OptionKey, object> PreferImplicitTypeEverywhere => OptionsSet(
```

Comment:
😕 Should this now be `Explicit` instead of `Implicit`?

Reply:
fixed.  should have been 'onWithInfo' anyways.